packages/sdk/src/signing.ts

```diff
@@ -17,19 +17,23 @@ export class Signing {
    * Sign Namada transaction
    * @param txProps - TxProps
    * @param signingKey - private key
+   * @param xsks - spending keys
    * @param [chainId] - optional chain ID, will enforce validation if present
    * @returns signed tx bytes - Promise resolving to Uint8Array
    */
   async sign(
     txProps: TxProps,
     signingKey: string,
+    xsks: string[],
     chainId?: string
```

Comment:
Quick question on `xsks: string[]` - would it be better to treat this as an optional parameter since this is specific to MASP?